client/src/components/chatComponents/displayMessages/singleChat/singleChat.css

```diff
@@ -0,0 +1,51 @@
+.message-container {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: auto auto;
+  gap: 5px 0px;
+  grid-auto-flow: row;
+  grid-template-areas:
+    "msg-sent-at"
+    "msg-box";
+}
+
+.msg-sent-at {
+  grid-area: msg-sent-at;
+}
+
+.msg-box {
+  grid-area: msg-box;
+}
+
+.message-container {
+  margin: 1em;
+}
+
+.right-pos {
+  float: right;
+}
+
+.left-pos {
+  float: left;
+}
+
```

Comment:
Usually better to use element intents so here sender and receiver or something will look better.